.github/ISSUE_TEMPLATE/depr-ticket.yml

```
@@ -28,16 +28,16 @@ body:
     id: transition-unblocked-date
     attributes:
       label: Target Transition Unblocked Date
-      description: When is the target date for having a replacement ready, and unblocking operators for preparing for the final breaking change? See [OEP-21](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html) for more details.
-      placeholder: "2025-03-15"
+      description: What is an estimated target date for unblocking operators for their transition tasks in preparation for any final breaking change? Where a replacement is required, the replacement must be ready. See [OEP-21](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html) for more details.
+      placeholder: "Estimated March or April 2025"
     validations:
       required: true
   - type: input
     id: breaking-change-unblocked-date
     attributes:
       label: Earliest Breaking Changes Unblocked Date
       description: When is the earliest date you may make breaking changes? A good default is the later of 6 months after the RFC Start Date, or 1 month after the Target Transition Unblocked Date. This date can be negotiated as part of the RFC process. See [OEP-21](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html) for more details.
-      placeholder: "2025-07-15"
+      placeholder: "Later of 2025-07-15 or the actual Transition Unblocked Date + 1 month"
```

Hmm, this seems a little jargoney to me... I'm trying to figure out how to make this easier to understand for devs who are not closely involved in the DEPR process.

```suggestion
      placeholder: "2025-07-05, or one month after the ticket moves to Transition Unblocked, whichever comes later."
```
IMO this is sounds a little more grokkable to me, but up to you if/how you want to incorporate it. Not a blocking suggestion.